pkg/model/resources/nodeup.go

```diff
@@ -149,6 +149,13 @@ function download-release() {
 
 /bin/systemd-machine-id-setup || echo "failed to set up ensure machine-id configured"
 
+{{ if RestartJournald -}}
+# DO cloud provider has /etc/machine-id baked in the image and 
+# systemd-machine-id-setup creates a new id and journalctld should be flushed
+# to use the new machine-id
+systemctl restart systemd-journald
+{{- end }}
+
```

Comment:
This adds extra spaces so it will require to run `hack/update-expected` to fix the golden output.

Reply:
`hack/update-expected` doesn't seem to work for me.. Trying few options here.